arm-documentdb/2015-04-08/documentdb.json

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
arm-documentdb/2015-04-08/**swagger/**documentdb.json. We missed a folder. Please take a look at other specs. They have a swagger folder inside the api-version folder.